frontend/src/helpers/question/question_api_wrappers.tsx

```diff
@@ -27,6 +27,7 @@ export async function getQuestionList(): Promise<Question[]> {
     method: HTTP_METHODS.GET,
     service: service,
     tags: scope,
+    cache: "no-cache",
```

Comment:
is there a reason why cache mode is set to no-cache?

Reply:
Hmm, I think NextJS somehow cache the wrong data? I have updated my database in backend, but it seems like NextJS are still showing the old database questions. And additionally, somehow without forcing the frontend to call the backend API, the page fails to load. Did you encounter this on your side?

Reply:
hmm... I did not try updating the database directly before, but the behavior is expected as NextJS uses caching by default. anyway, I'm fine with disabling the cache since the question service is meant for admins only.

Reply:
NextJS caching is meant for frontend only. Redis is more for backend caching.... 
I don't see any conflict here, unless you meant to use Redis for other purposes?

Reply:
Oh ya, you are right. I think what I meant is to disable NextJS caching for some cases (I think especially for getUser) and implement Redis cache in the backend API instead. Not sure if the questions part need the caching? But I don't think we need to do that so soon. This is something like a good-to-have haha.

Reply:
Ya, I guess each service has to decide when and if they want to use NextJS caching, so I leave the caching option and scope open to them.